jobserver/github.py

```diff
@@ -73,6 +73,35 @@ def get_file(org, repo, branch):
     return r.text
 
 
+def get_repo(org, repo):
+    f = furl(BASE_URL)
+    f.path.segments += [
+        "repos",
+        org,
+        repo,
+    ]
+
+    headers = {
+        "Accept": "application/vnd.github.v3+json",
+        "Authorization": f"token {GITHUB_TOKEN}",
+        "User-Agent": USER_AGENT,
+    }
+    r = requests.get(f.url, headers=headers)
```

Comment:
Fine as is for now, but this might benefit from splitting out the generic "talk to github API" code from the stuff specific to repos at some point.

Reply:
Yeah, I've been thinking about this.  We have a couple of small differences so far (one gets a file, one is the GraphQL paging stuff…), but something for the bog standard "get some JSON please" is definitely starting to look useful!